eggbox_potential_sampler/random_sampling_mco/mco.py

```diff
@@ -35,11 +35,11 @@ def run(self, model):
         application = self.factory.plugin.application
         if model.evaluation_mode == "Subprocess":
             single_point_evaluator = SubprocessSinglePointEvaluator(
-                sys.argv[0], application.workflow_filepath
+                sys.argv[0], application.workflow_file.path
```

Comment:
It seems right now that only the `application.workflow_file` is used in `.run`. Is it possible to replace the line 35 to 
```
workflow_file = self.factory.plugin.application.workflow_file
```
and use it straight away?